src/current/v23.1/alter-table.md

```diff
@@ -911,6 +911,50 @@ $ cockroach demo bank
 (1 row)
 ~~~
 
+##### Move a column from one column family to another
+
+To move a column from one column family to another column family, create a temporary, [non-visible](create-table.html#not-visible-property) [stored computed column](computed-columns.html) in the target column family, then rename the columns. Once this succeeds, you can drop the original, now renamed column.
```

Comment:
First use of 'column family' could be a link to our overview docs on the subject at https://www.cockroachlabs.com/docs/stable/column-families.html